vaccine_feed_ingest_schema/location.py

```diff
@@ -203,6 +196,36 @@ class VaccineProvider(str, enum.Enum):
     PAVILIONS = "pavilions"
     WALMART = "walmart"
     CVS = "cvs"
+    ACME = "acme"
+    BIG_Y = "big_y"
+    BROOKSHIRE = "brookshire"
+    COSTCO = "costco"
+    CUB = "cub"
+    DILLONS = "dillons"
+    DRUGCO = "drugco"
+    FAMILY_FARE = "family_fare"
+    FOOD_CITY = "food_city"
+    FOOD_LION = "food_lion"
+    FRED_MEYER = "fred_meyer"
+    FRYS = "frys"  # Frys Food and Drug
+    GENOA = "genoa"  # Genoa Healthcare LLC
```

Comment:
Yeah, I added those comments because the current convention around naming is a bit unclear to me. For example, there is a `Cub Pharmacy`, for which I am using `cub` as the enum, following the convention of `Walgreen's Pharmacy` using `walgreens`. But maybe `cubs_pharmacy` will work just as well. In general, choosing a more verbose enum for lesser known brands cannot hurt,  I guess ... I'll update the enums for the ones you suggested, and also take a closer look at the other ones I've added to see if there is opportunity to improve them accordingly.

Reply:
This is a good question and I don't have a great answer. The uncertainty about this makes me especially glad that we have the enum!

I agree with @shashank025 that the distinction is something like: how likely is a random person working with the code/data (as opposed to an end-user) to have heard of the location? So, I'd be inclined to go with `cub_pharmacy` bu and `frys_food_and_drug`, but `walgreens` and probably even `heb`.

Overall I don't think this matters _too_ much thanks to the enum, so I don't think that we need to get it precisely right in edge cases.

Reply:
OK, that aligns with my thinking as well! Sending an update shortly.